app/schemas/v1/document/report_type.json

```diff
@@ -0,0 +1,38 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+
+    "report_type": {
```

Comment:
The plan is to store the report_type and subjets in the database. We will need to figure out a way to dynamically populate the values.